recipe/meta.yaml

```
@@ -31,7 +31,7 @@ requirements:
     - pkg-config
   host:
     - python
-    - cython 0.29
+    - cython 3.0.0
```

```suggestion
    - cython >=3.0.0,<3.1.0
```
To make our life easier with rebuilds.
https://github.com/astropy/astropy/blob/v6.1.0/pyproject.toml#L128C20-L128C34